README.md

```diff
@@ -48,50 +48,42 @@ allprojects {
 }
 ```
 
-### 3. Add JNA library dependency
+### 3. Download Android libindy binaries
```

Comment:
Do you have any instructions on how to build the `.aar` file? Otherwise could we also keep the non-aar instructions in here?

Reply:
Yes, you can do it in the following way:

- In Android Studio, click File > New > New Module and select Android Library.
- The simplest way to add so files into an aar file is to create a folder `jniLibs` under `src/main` And then copy `*.so` files into the folder:
```
src/main/jniLibs/<abi>/*.so
```
- After building the module, you can open `build/outputs/aar/*.aar` to verify `jni//*.so`.
- Resources
  - [Create an Android Library (AAR). Steps to create an Android library… | by Anusree Sajeevan | ProAndroidDev](https://proandroiddev.com/create-an-android-library-aar-79d2338678ba)
  - [Different Ways to Create aar File in Android Studio - GeeksforGeeks](https://www.geeksforgeeks.org/different-ways-to-create-aar-file-in-android-studio/)
  - [How to Build *.so Library Files into AAR Bundle in Android Studio | Dynamsoft Developers](https://www.dynamsoft.com/codepool/build-so-aar-android-studio.html)